osgp/platform/osgp-domain-core/src/main/java/org/opensmartgridplatform/domain/core/valueobjects/smartmetering/GetOutagesRequestData.java

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright 2015 Smart Society Services B.V.
```

Comment:
This file looks like it is new. In that case the copyright has the wrong name and year.